ui/imports/shared/controls/FoldableHeader.qml

```diff
@@ -55,7 +55,7 @@ Rectangle {
             id: modeSwitch
             objectName: "switch"
 
-            visible: !!text
+            visible: !!text && !!root.ListView.view && root.ListView.view.model && root.ListView.view.model.count
```

Comment:
Why this extra condition is needed?

Reply:
It was showing the "Arrange by..." switches even when the model was empty, which didn't make sense